src/test/util/logging.cpp

```diff
@@ -21,11 +22,12 @@ DebugLogHelper::DebugLogHelper(std::string message, MatchFn match)
     noui_test_redirect();
 }
 
-void DebugLogHelper::check_found()
+DebugLogHelper::~DebugLogHelper()
 {
     noui_reconnect();
     LogInstance().DeleteCallback(m_print_connection);
     if (!m_found && m_match(nullptr)) {
-        throw std::runtime_error(strprintf("'%s' not found in debug log\n", m_message));
+        std::cerr << "Fatal error: expected message not found in the debug log: " << m_message << "\n";
```

Comment:
The original error had the message string in quotes. I think this should be preserved because it is easier to debug when the string contains whitespace at the end.

Reply:
```suggestion
        tfm::format(std::cerr, "Fatal error: expected message not found in the debug log: '%s'\n", m_message);
```

Reply:
Done, thanks!